tests/e2e/profile-happy-path.test.ts

```diff
@@ -42,7 +43,7 @@ describe.only("Profile Happy path", () => {
   ).includes("staging");
 
   test("it should do Profile and Addresses flows' happy path", async () => {
-    expect.assertions(isStagingEnv ? 29 : 21);
+    expect.assertions(isStagingEnv ? 29 : 19);
```

Comment:
```suggestion
    expect.assertions(isStagingEnv ? 28 : 19);
```
If I'm not wrong, you delete two expects, one inside a non-staging env case and one that occurs for both cases.

Reply:
Oh you are right, good catch!